neurodamus/target_manager.py

```diff
@@ -282,9 +284,17 @@ class NodeSetReader:
     Implements reading Sonata Nodesets
     """
 
-    def __init__(self, nodeset_file):
-        self.nodesets = libsonata.NodeSets.from_file(nodeset_file)
+    def __init__(self, config_nodeset_file, simulation_nodesets_file):
+        def _load_nodesets_from_file(nodeset_file):
+            if not nodeset_file:
+                return libsonata.NodeSets("{}")
```

Comment:
Just to be sure, if the path is defined it must exist, otherwise it's an error?

Reply:
Yes, but it was the same before right?
```
self.nodesets = libsonata.NodeSets.from_file(nodeset_file)
```